domains/sam-jspn.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Sam's website",
+  "owner": {
+    "username": "Sam-Jspn",
+    "email": "sam.josephine@outlook.fr",
+    "discord": "R3dL1ghty#2904"
+  },
+  "record": {
+    "CNAME": "https://samuel-jspn.github.io/lair/"
```

Comment:
cname shouldnt contain slash

Reply:
should it just be "samuel-jspn.github.io/lair" ? Or no slash at all ?

Reply:
no slash.